### PR TITLE
Added jobs validation

### PR DESCRIPTION
**Pull request by AndyObtiva**
Implemented https://github.com/montrealrb/Montreal.rb/issues/93 adding job model/form validations

[Review on Reviewable](https://reviewable.io/reviews/montrealrb/montreal.rb/136)

### CODE REVIEW

File: app/models/job.rb [hunk 1]
```diff
@@ -1,6 +1,11 @@
 class Job < ActiveRecord::Base
   extend Enumerize
-  STATUSES = [:draft, :published, :archived]
+  STATUSES = %w(draft published archived)
   enumerize :state, in: STATUSES
```
Comment (cawel) [inline app/models/job.rb]: Smell: we're using 2 words ('status' and 'state') to refer to the same concept.

Maybe we could simply say this?

```ruby
STATES = %w(draft published archived)
enumerize :state, in: STATES
```

Comment (AndyObtiva) [inline app/models/job.rb]: Good point. I just revised it to say STATES.

**Comment by nicholasjhenry** [comment]
Just a heads up @AndyObtiva that this requires a rebase. Cheers!

**Comment by AndyObtiva** [comment]
OK, rebased from trunk.

**Comment by steakunderscore** [comment]
@AndyObtiva Mind if I update this patch? Or would you be able to? I was also wanting to use shoulda-matchers.

**Comment by AndyObtiva** [comment]
Go ahead if you like. I can't stop you if GitHub gives you and everybody access.

**Comment by steakunderscore** [comment]
@AndyObtiva Thanks, but I still don't want to be stepping on ya toes!

**Comment by sophiedeziel** [comment]
fixed by #177

**Pull request closed without merging**